hoot-core-test/src/test/cpp/hoot/core/ops/DuplicateWayRemoverTest.cpp

```diff
@@ -166,8 +157,7 @@ class DuplicateWayRemoverTest : public CppUnit::TestFixture
   }
 };
 
-}
-
 CPPUNIT_TEST_SUITE_NAMED_REGISTRATION(DuplicateWayRemoverTest, "quick");
 //CPPUNIT_TEST_SUITE_NAMED_REGISTRATION(DuplicateWayRemoverTest, "current");
 
+}
```

Comment:
This is a good example of the class being in the `hoot` namespace while the `CPPUNIT_TEST_SUITE_NAMED_REGISTRATION` wasn't.